SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -12,6 +10,30 @@ DEFINE_LOG_CATEGORY(LogSpatialEventTracer);
 
 namespace SpatialGDK
 {
+TraceQueryPtr ParseOrDefault(const FString& Str, const char* FilterForLog)
+{
+	TraceQueryPtr Ptr;
+	if (Str.Len())
```

Comment:
Omega nit: does our coding standard say anything about casting int to bool?

Reply:
It does, my bad - will fix!

Reply:
our coding standard is very tiny, it doesn't cover this

Reply:
Epic's doesn't either.